pkgs/desktops/gnome-3/3.20/core/libpeas/default.nix

```diff
@@ -1,13 +1,15 @@
 { stdenv, fetchurl, pkgconfig, intltool, gnome3
-, glib, gtk3, gobjectIntrospection, python, pygobject3
+, glib, gtk3, gobjectIntrospection, python3, python3Packages, ncurses, python2, python2Packages
 }:
 
 stdenv.mkDerivation rec {
   inherit (import ./src.nix fetchurl) name src;
 
+  configureFlags = [ "--enable-python3" "--enable-python2" ];
+
   buildInputs =  [
-   intltool pkgconfig glib gtk3 gobjectIntrospection python pygobject3
-   gnome3.defaultIconTheme
+   intltool pkgconfig glib gtk3 gobjectIntrospection python3 python3Packages.pygobject3 python2
+   gnome3.defaultIconTheme ncurses
```

Comment:
@FRidh I would like to merge this PR as it is (it works now). I don't have enough energy now to keep searching for the solution. Is it ok with you?